### PR TITLE
Add Grafana dashboard

### PR DESCRIPTION
**Pull request by kelkawi-a**
This PR adds a Grafana dashboard for monitoring infrastructure and application metrics captured from Airbyte.

### CODE REVIEW

File: opentelemetry/grafana_dashboard.json [hunk 1]
```diff
@@ -0,0 +1,2159 @@
...
+          "interval": "",
+          "legendFormat": "",
+          "refId": "A"
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeRegions": [],
+      "timeShift": null,
+      "title": "Average CPU usage per pod",
```
Comment (gtato) [inline opentelemetry/grafana_dashboard.json]: what is the unit of this?
also... is it possible to have this as a percentage and then have also the total in the same graph?

Comment (gtato) [reply opentelemetry/grafana_dashboard.json]: maybe we can have the same for memory... so maybe we can have just 3 graphs for the infrastructure metrics:

cpu usage per pod (including total) in %
memory usage per pod (including total) in %
network stuff

Comment (kelkawi-a) [reply opentelemetry/grafana_dashboard.json]: I've updated to include the units `seconds/second`. Having them on the same chart might not be practical given the difference in value range. This chart will mainly be used to identify if any pods are consuming too much CPU.

I've updated the memory usage chart to also include per pod memory usage (%), and updated this chart to be per pod cpu usage (%).

As the PromQL queries are slightly different, I've chose not to combine them.

Comment (gtato) [reply opentelemetry/grafana_dashboard.json]: if everything is in % why would they have different ranges?

Comment (kelkawi-a) [reply opentelemetry/grafana_dashboard.json]: Sorry I made two changes at the same time and confused them. The infrastructure charts have been updated to the following:
- CPU usage of namespace (taking into account number of cores in cluster).
- CPU usage per pod and total CPU usage (compared to namespace)
- Memory usage (total and per pod percentage)
- Maximum memory usage
- Namespace memory usage %
- Network receive and transmit

**Review by gtato — APPROVED** [review]

**Review by kian99 — APPROVED** [review]
I can't review each line but I'm assuming you verified the dashboard. Out of curiosity, how is this dashboard passed to Grafana?

**Review by mertalpt — COMMENTED** [review]
I think this is one of the cases where a quick call to present the changes would work better. Also, 2nd the question about how this gets imported to Grafana.

**Comment by kelkawi-a** [comment]
> I can't review each line but I'm assuming you verified the dashboard. Out of curiosity, how is this dashboard passed to Grafana?

The dashboard can be created on Grafana manually. I've only included the dashboard definition here for clarity.

**Pull request merged**